### ISSUE TITLE
Migrate Box layouts to Grid based ones 

### ISSUE THREAD

**Issue by elbenfreund** [projecthamster/hamster-gtk#27]
According to [the docs](https://developer.gnome.org/gtk3/stable/GtkVBox.html) ``Gtk.Box`` whilst not deprecated is advised against in favor of all out ``Gtk.Grid`` based layouts. For migration tips [refer to this](https://developer.gnome.org/gtk3/stable/gtk-migrating-GtkGrid.html).

This should not be a big deal and is of rather low priority right now, but should be done before the first proper release none the less.
